util/tracer_nvbit/tracer_tool/traces-processing/post-traces-processing.cpp

```diff
@@ -104,6 +104,9 @@ void group_per_block(const char *filepath) {
   string string1, string2;
   bool found_grid_dim = false, found_block_dim = false;
 
+  // Ni: Add a flag for LDGSTS instruction to indicate which one to remove
+  vector<vector<bool>> LDGSTS_flag;  // true to remove, false to not

```

Comment:
it's better to keep the variable naming convention consistent, so don't use upper case - "ldgsts_flags" is better